code_map.py

```diff
@@ -866,7 +872,7 @@ def on_text_command(self, view, command_name, args):
 
     def on_window_command(self, window, command_name, args):
         '''Very unstable switching projects, safety measure'''
-        global TEMP_VIDS, TEMP_VIEWS, CURRENT_TEMP_ID, ACTIVE
+        global TEMP_VIDS, TEMP_VIEWS, CURRENT_TEMP_ID, ACTIVE, PAUSED
 
         reset = ["prompt_open_project_or_workspace",
                  "prompt_switch_project_or_workspace",
```

Comment:
when switching project CodeMap is paused until new files are loaded, so its listeners don't work